endpoints/openrtb2/auction.go

```diff
@@ -1980,6 +2007,73 @@ func getAccountID(pub *openrtb2.Publisher) string {
 	return metrics.PublisherUnknown
 }
 
+// Returns the account from the request
```

Comment:
Nitpick: Please follow Go's comment standards for this method. ie:

```
// getAccountIdFromRawRequest returns the account from the request
```

. or if you'd like to omit the comment entirely, the method name is self explanatory IMHO.